changelogs/DP-38400.yml

```
@@ -0,0 +1,6 @@
+Changed:
+  - project: Patternlab
+    component: Key Message
+    description: Adjusts the font sizing of the subtitle. Adds additional padding around the container. Adds a new variant for two columns of text.
```

```suggestion
    description: Adjusts the font sizing of the subtitle. Adds additional padding around the container. Adds a new variant for two columns of text. (#1969)
```
Just a note, we add PR number to the description